### PR TITLE
fixing exemplar similarity function

### PR DESCRIPTION
**Pull request by tfjaeger**
Made some minor corrections to the definition of exemplar models

### CODE REVIEW

File: R/make-objects.R [hunk 1]
```diff
@@ -243,11 +243,16 @@ make_exemplar_model_from_data = function(
...
     category = "category",
     cues,
-    sim_function = function(x, y) {
-      j <- 2
-      k <- 1
-      distance <- (x - y)^j
+    sim_function = function(x, y, j = 2, k = 1) {
+      # If j = 1, this uses Manhattan city block distances
+      # If j = 2, this uses Euclidean distances
+      distance <- sum((x - y)^j)^-j
```
Comment (sultan8) [inline R/make-objects.R:249]: just checking: this expression gives 1/(sum of distances raised by j)

Comment (sultan8) [reply R/make-objects.R:249]: I meant line 249

Comment (tfjaeger) [reply R/make-objects.R:249]: Good question. Not quite. What you wrote would be sum((x-y)^j)^-1.

But I made mistake. What I want is the generalization of the formula for the Euclidean distance:

sqrt(sum((x - y)^j)) = sum((x - y)^j)^(1/2)

So, it should be:

sum((x - y)^j)^(1/j). I'll push a fix. Can you look over it again and then approve if all looks good?

File: R/make-objects.R [hunk 2]
```diff
@@ -243,11 +243,16 @@ make_exemplar_model_from_data = function(
...
     category = "category",
     cues,
-    sim_function = function(x, y) {
-      j <- 2
-      k <- 1
-      distance <- (x - y)^j
+    sim_function = function(x, y, j = 2, k = 1) {
+      # If j = 1, this uses Manhattan city block distances
+      # If j = 2, this uses Euclidean distances
+      distance <- sum((x - y)^j)^(1/j)
```
Comment (sultan8) [inline R/make-objects.R:249]: I just realised that Euclidean distance is just the Pythagorean distance. If I had known this I wouldn't have had to be so mystified all this time.

**Review by sultan8 — APPROVED** [review]

**Pull request merged**